### PR TITLE
feat: accounts orders

### PR DESCRIPTION
**Pull request by FabianGerke**
## Why:

Closes: 
[PRODUCT BACKLOG ITEM 71440](https://dev.azure.com/plentymarkets/plentymarkets/_workitems/edit/71440)

## Checklist before requesting a review
- [ ] My code follows the code style of this project.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/composables/useCustomerOrders/useCustomerOrders.ts [hunk 1]
```diff
@@ -27,20 +22,15 @@ export const useCustomerOrders: UseCustomerOrdersReturn = () => {
    * @example
    * fetchCustomerOrders();
```
Comment (maxiroellplenty) [inline apps/web/composables/useCustomerOrders/useCustomerOrders.ts:23]: add example for the fetchCustomerOrders payload

File: apps/web/pages/my-account/my-orders.vue [hunk 2]
```diff
@@ -1,89 +1,133 @@
...
+
+const setMaxVisiblePages = (isWide: boolean) => (maxVisiblePages.value = isWide ? 5 : 1);
+const route = useRoute();
+
+watch(isWideScreen, (value) => setMaxVisiblePages(value));
+onMounted(() => setMaxVisiblePages(isWideScreen.value));
+
 const { fetchCustomerOrders, data } = useCustomerOrders();
-await fetchCustomerOrders();
+// const NuxtLink = resolveComponent('NuxtLink');
```
Comment (maxiroellplenty) [inline apps/web/pages/my-account/my-orders.vue:117]: whats up with that?

File: apps/web/pages/my-account/my-orders.vue [hunk 3]
```diff
@@ -1,89 +1,133 @@
 <template>
```
Comment (maxiroellplenty) [inline apps/web/pages/my-account/my-orders.vue:1]: we might want to have a loading spinner or disable the pagination while the data is loading. The calls take some time and if you are too fast you might get a call thats not for the page :D

**Review by maxiroellplenty — DISMISSED** [review]

File: apps/web/pages/my-account/my-orders.vue (hunk 2 above)
Comment (FabianGerke) [reply apps/web/pages/my-account/my-orders.vue:117]: Is needed for the "Details" implementation

Comment (maxiroellplenty) [reply apps/web/pages/my-account/my-orders.vue:117]: ok 👍

**Review by abocsan-plenty — APPROVED** [review]

**Pull request merged**